run.py

```diff
@@ -177,10 +182,20 @@ def gh_mime_data(app):
     garb_handler_md.remove(app)
 
 
+def cache_data(app):
+    all_titles = list()
+    all_results = list()
+    w = walker.Walker(app)
+    for achievement in w.get_all_achievements():
+        all_titles.append(achievement.container.title)
+        all_results.append(achievement.result)
+
+
 def timeout_daily(app):
     log.info("daily execution handler started")
     gh_container_achievements(app)
     gh_mime_data(app)
+    cache_data(app)
```

Comment:
In the second code block for cache_tags.py. Is there really an old cache_tags.db to open?
Isn't it more like we write always a new file, when the get_all_achievements() process starts.
Because if we, e.g. would allow multiple entries of the same tag/category to get some statistics about how often the tag/category is used and then update the old file...wouldn't it distort the statistic.
Otherwise if we wouldn't allow multiple entries of the same tag/category...the updating process of an old file is unproblematic.
So the question: multiple entries of same tag/category...yes/no?
If yes ---> always write new cache_tags.db file when get_all_achievements() starts?